analytics/views/remote_activity.py

```diff
@@ -78,28 +102,60 @@ def get_remote_server_activity(request: HttpRequest) -> HttpResponse:
     LAST_AUDIT_LOG_DATE = 4
     MOBILE_USER_COUNT = 5
     MOBILE_PUSH_COUNT = 6
+    REALM_ID = 7
+    ORG_TYPE = 9
+    ARR = 12
+    TOTAL_USER_COUNT = 13
+    GUEST_COUNT = 14
 
     rows = get_query_data(query)
-    total_row = []
     plan_data_by_remote_server = get_plan_data_by_remote_server()
+    plan_data_by_remote_realm = get_plan_data_by_remote_realm()
+
+    total_row = []
+    server_mobile_data_counted = []
```

Comment:
This should be a set, right? Otherwise this algorithm is quadratic in the number of servers.

Reply:
Pushed update so this is a set and not a list. Also updated variable name to be clearer that it's remote servers.